dashboard/config/locales/slides.en.yml

```diff
@@ -2451,3 +2451,76 @@ en:
       9:
         image: 'notes/mc_2018_closing_9.png'
         text: "You've certainly proven yourself an intrepid coder by now, and I can’t wait to see what you build. And hopefully your adventures with code don't end here. Bon Voyage!"
+    'Teaching CS Fundamentals: How the Course Works':
```

Comment:
Are we still looking for a way to make this snake_case?

Reply:
Yes, the PR will be linked.

Reply:
PR: https://github.com/code-dot-org/code-dot-org/pull/27740. You'll want to merge staging into this branch after merging that PR, and update the string here as well 👍.